xml2rfc/writers/text.py

```diff
@@ -2150,7 +2150,9 @@ def render_li(self, e, width, **kwargs):
         tt, __ = self.text_or_block_renderer(e, width, **kwargs)
         if isinstance(tt, list):
             lines = stripl(tt)
-            if lines and lines[0].elem.tag not in ['artwork', 'figure', 'sourcecode', 'li', ]:
+            if (lines and
```

Comment:
is this what black would produce?

Reply:
Updated with https://github.com/ietf-tools/xml2rfc/pull/1137/commits/fdb6fb8ad2024b3483f7480a07f8370e673c27e2